content/_data/events/2018-12-18-jenkins-online-meetup.adoc

```diff
@@ -0,0 +1,7 @@
+---
+title: "Jenkins Online Meetup"
+date: “2018-12-18T17:00:00”
+link: "https://www.meetup.com/Jenkins-online-meetup/events/257008190/"
+---
+Java 11 Support preview.
+We will show how to run Jenkins with Java 11, discuss known compatibility issues, and explain how to investigate and report them.
```

Comment:
Nit: possibly not clear enough it's not for production usage yet with this sentence, but well the title should help I think.